__tests__/endpoint.test.js

```diff
@@ -259,4 +259,26 @@ describe("Endpoint Tests", () => {
           .expect(404)
       });
 })
+describe("GET /api/users", () => {
+  test("200: returns all data from users when requesting all data", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        body.data.forEach((users) => {
+          expect(typeof users.username).toBe("string");
+          expect(typeof users.name).toBe("string");
+          expect(typeof users.avatar_url).toBe("string");
+        });
+      });
+  });
```

Comment:
Nice, but dont forget to have an assertion to ensure the array contains some elements. Just so an empty array cannot pass this test